OrbitGl/App.cpp

```diff
@@ -954,13 +954,14 @@ bool OrbitApp::GetSamplingEnabled() { return GParams.m_TrackSamplingEvents; }
 
 //-----------------------------------------------------------------------------
 
-void OrbitApp::OnProcessSelected(uint32_t pid) {
-  thread_pool_->Schedule([pid, this] {
+void OrbitApp::OnProcessSelected(uint32_t pid,
+                                 const std::string& process_name) {
+  thread_pool_->Schedule([pid, &process_name, this] {
```

Comment:
I am not sure reference here is safe. the caller is not obligated to keep it after the method call. I think copy is safer.

Reply:
This is now outdated, but you are right.